permissions/plugin-zoom.yml

```diff
@@ -0,0 +1,7 @@
+---
+name: "zoom"
+github: "jenkinsci/zoom-plugin"
+paths:
+- "org/jenkins-ci/plugins/zoom"
```

Comment:
Wrong path. It should match the plugin's groupId. I.e. `io.jenkins.plugins`

Reply:
@batmat Sorry, is that all right now?